### PR TITLE
Adding mention of trigger and Tik Manager

### PR DESCRIPTION
**Pull request by masqu3rad3**
Trigger is a modular rigging and automation tool for Autodesk Maya.
https://github.com/masqu3rad3/trigger

Tik Manager is an open source python based multi DCC and cross-platform project & asset management software.
The primary objective of Tik Manager 4 is to maintain simplicity for small projects while offering the flexibility needed to handle more complex and larger-scale endeavors.
https://github.com/masqu3rad3/tik_manager4

### CODE REVIEW

**Review by stilobique — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -213,6 +213,7 @@ Any contribution is welcome!
 * [qLib](https://github.com/qLab/qLib) - A procedural asset library for SideFX Houdini.
 * [SideFX Labs](https://github.com/sideeffects/SideFXLabs) - A completely free, open-source toolset geared towards assisting Houdini users with a variety of tasks commonly used for digital content creation.
 * [xComp](https://github.com/gugenstudio/xComp) - Image Render comparing.
+* [trigger](https://github.com/masqu3rad3/trigger) - Operational code driven rigging tool for Autodesk Maya.
```
Comment (darkvertex) [inline README.md:216]: Can  you move it up a line? We try to keep stuff alphabetical and T comes before X.

**Review by darkvertex — CHANGES_REQUESTED** [review]
Fix the alphabetical ordering. Otherwise LGTM!

File: README.md (hunk 1 above)
Comment (masqu3rad3) [reply README.md:215]: Sure thing. Made another commit.

**Review by darkvertex — APPROVED** [review]

**Comment by frankrousseau** [comment]
Thank you @masqu3rad3!

**Comment by masqu3rad3** [comment]
Thank you @frankrousseau  and @darkvertex

**Pull request merged**